### PR TITLE
Tabs and formatting

### PR DESCRIPTION
**Pull request by pthariensflame**

### CODE REVIEW

File: SwiftCheck/Property.swift [hunk 1]
```diff
@@ -197,14 +203,16 @@ extension Testable {
...
-			return TestResult(ok: res.ok,
-						expect: false,
-						reason: res.reason,
-						theException: res.theException,
-						labels: res.labels,
-						stamp: res.stamp,
-						callbacks: res.callbacks,
-						abort: res.abort)
+			return TestResult(
+				ok: res.ok,
```
Comment (CodaFi) [inline SwiftCheck/Property.swift]: These need to be put back.  They're indented very specifically.

File: SwiftCheck/Property.swift [hunk 2]
```diff
@@ -279,19 +289,19 @@ public enum CallbackKind {
 
 public enum TestResultMatcher {
 	case MatchResult( ok : Optional<Bool>
-					, expect : Bool
-					, reason : String
-					, theException : Optional<String>
```
Comment (CodaFi) [inline SwiftCheck/Property.swift]: This too.

File: SwiftCheck/Property.swift [hunk 3]
```diff
@@ -363,7 +373,16 @@ private func props<A>(shrinker : A -> [A], original : A, pf: A -> Testable) -> R
 }
 
 private func result(ok : Bool?, reason : String = "") -> TestResult {
-	return TestResult(ok: ok, expect: true, reason: reason, theException: .None, labels: [:], stamp: Set(), callbacks: [], abort: false)
+	return TestResult(
+		ok: ok,
+		expect: true,
```
Comment (CodaFi) [inline SwiftCheck/Property.swift]: And this

File: SwiftCheck/Test.swift [hunk 4]
```diff
@@ -392,58 +392,58 @@ internal func findMinimalFailingTestCase(st : CheckerState, res : TestResult, ts
...
+		, computeSize: st.computeSize
+		, numSuccessTests: st.numSuccessTests
+		, numDiscardedTests: st.numDiscardedTests
+		, labels: st.labels
+		, collected: st.collected
+		, expectedFailure: st.expectedFailure
+		, randomSeed: st.randomSeed
+		, numSuccessShrinks: numSuccessShrinks
+		, numTryShrinks: numTryShrinks
+		, numTotTryShrinks: numTotTryShrinks
```
Comment (CodaFi) [inline SwiftCheck/Test.swift]: And this

**Comment by pthariensflame** [comment]
@CodaFi How does that look now?

File: SwiftCheck/Testable.swift [hunk 5]
```diff
@@ -46,11 +46,11 @@ public struct Property : Testable {
 /// A proposition.
 public struct Prop : Testable {
 	var unProp : Rose<TestResult>
-	
+
 	public var exhaustive : Bool { return true }
 
 	public var property : Property {
-//		return Property(Gen.pure(Prop(unProp: .IORose(protectRose({ self.unProp })))))
+		//		return Property(Gen.pure(Prop(unProp: .IORose(protectRose({ self.unProp })))))
```
Comment (CodaFi) [inline SwiftCheck/Testable.swift]: Put that back

**Comment by CodaFi** [comment]
One last little thing and then we're good to go

File: SwiftCheck/Testable.swift (hunk 5 above)
Comment (pthariensflame) [inline SwiftCheck/Testable.swift]: Ah, how could I have forgotten?  :)

**Comment by pthariensflame** [comment]
:grey_question:

**Comment by CodaFi** [comment]
:exclamation: 

:boat:

**Pull request merged**